src/app/shared/components/search/search.component.html

```diff
@@ -1,8 +1,16 @@
 <div class="search">
   <div>
-    <span class="img-search"><img src="assets/img/ic-search@2x.png" class="ic_search"></span>
-    <input type="search" placeholder="Text to search"/>
-    <button type="button" class="btn btn-default sick-green">Search</button>
+    <span class="img-search"
+      ><img src="assets/img/ic-search@2x.png" class="ic_search"
+    /></span>
```

Comment:
/> should be on the previous line and <span> on the next line. 
You should have smth like :
`<span>`
`<img />`
`</span>`